### PR TITLE
Feature/v1-028 Detailed course page

### PR DESCRIPTION
**Pull request by hauteballe**
**Overview:**
1. Implement Detailed course page with default data from constants folder
2. Implement Similar courses section with data from the server
2. Implement ProgressBar component with default data (will be changed to percentage and status from the sever )
3. Add useApplyCourse hook 
4. Button "Start the course" is redirecting to "My courses" page now (will be changed after Learning course page will be merged)
5. Fixed an issue with Help page scrollbars

### CODE REVIEW

File: frontend/src/pages/CoursesList/CoursesContainer.tsx [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+import useApplyCourse from 'api/courses/applyCourse';
+
 import CoursesList from './CoursesList';
 
 const CoursesContainer: React.FC = () => {
   const { data, isLoading } = useGetCourses();
 
-  return <CoursesList data={data} isLoading={isLoading} />;
+  const params = useParams();
+  const mutation = useApplyCourse();
```
Comment (k-krasilnikova) [inline frontend/src/pages/CoursesList/CoursesContainer.tsx:13]:
```suggestion
  const { mutate, isLoading: isLoadingApplying } = useApplyCourse();
```

File: frontend/src/pages/DetailedCourse/DetailedCourseContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import useGetCourseInfo from 'api/courses/getCourseInfo';
+import useApplyCourse from 'api/courses/applyCourse';
+
+import DetailedCourse from './DetailedCourse';
+
+const DetailedCourseContainer: React.FC = () => {
+  const params = useParams();
+  const { data } = useGetCourseInfo(params._id);
+  const mutation = useApplyCourse();
```
Comment (k-krasilnikova) [inline frontend/src/pages/DetailedCourse/DetailedCourseContainer.tsx:12]: We should de from the start what the props here:
```suggestion
  const { mutate, isLoading: isLoadingApplying } = useApplyCourse();
```

File: frontend/src/pages/DetailedCourse/DetailedCourseContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  const handleApplyCourse = () => {
+    mutation.mutate(params._id);
```
Comment (k-krasilnikova) [inline frontend/src/pages/DetailedCourse/DetailedCourseContainer.tsx:14]: Let's rename `_id` to the `courseId` around all the app

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]
Pretty good job, but some changes required!

**Review by Julia-Maliutsina — DISMISSED** [review]

**Pull request merged**